packages/components/Callout/src/Callout.tsx

```diff
@@ -14,14 +15,23 @@ export const Callout = compose<ICalloutType>({
   usePrepareProps: (props: ICalloutProps, useStyling: IUseComposeStyling<ICalloutType>) => {
     const { componentRef, target, ...rest } = props;
     const calloutRef = useViewCommandFocus(componentRef);
-    const targetNativeTag = findNodeHandle(target.current);
+    const [targetNativeTag, setTargetNativeTag] = React.useState<number>(null);
+
+    React.useLayoutEffect(() => {
+      if ((target !== undefined && target?.current !== null))
+      {
+        setTargetNativeTag(findNodeHandle(target.current));
+      }
+    }, [target]);
```

Comment:
Personally, I'm a fan of the more succinct JS syntax:

`if (target && target.current)`

This makes sure that `target` and `target.current` are both not falsy, which excludes both null and undefined.

Worth noting that `undefined !== null` is a true statement, so you will call findNodeHandle with an `undefined` value in `target.current` and `target` isn't guaranteed to be non-null after the first undefined check before the `&&`. I'm sure that function checks for undefined, but I'd probably avoid the entire if statement to begin with.  A `null` target might crash as is.

Another common pattern is just `target && target.current && setTargetNativeTag(findNodeHandle(target.current));`

Part of me wonders if this would be beneficial in a shared hook... but I suppose we could always move it when we need it :)